jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  testMatch: ['**/spec/**/*Spec.js'],
```

Comment:
Do you plan on migrating from spec -> test? Thought I saw a thread in slack about this

Reply:
You got it!